domain/snyk/scanner.go

```diff
@@ -69,7 +69,12 @@ func (sc *DelegatingConcurrentScanner) Scan(
 	done := make(chan bool)
 	defer close(done)
 
-	sc.initializer.Init()
+	err := sc.initializer.Init()
+	if err != nil {
+		log.Error().Err(err).Msg("Scan initialization error, cancelling scan")
```

Comment:
would logging to Sentry make sense here?

Reply:
The initializers report to Sentry so I don't think there's a need